examples/creating_printing_table/wscript_build

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env python
+# encoding: utf-8
+
+bld.program(
+    features='cxx',
+    source=['creating_printing_table.cpp'] + bld.path.ant_glob('src/*.cpp'),
```

Comment:
I think this line is wrong here. Let's remove the second part: `bld.path.ant_glob('src/*.cpp')`